linux-cmd-intallation.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Error handling
+trap 'last_command=$current_command; current_command=$BASH_COMMAND' DEBUG
+terminate()
+{
+    echo >&2 '
+*** terminated ***
+'
+    echo "\"$last_command\" command failed with exit code $?."
+    exit 1
+}
+
+trap 'terminate' 0
+
+set -e
+
+# clone repo
+
+# install required packages
+sudo dnf install subversion
```

Comment:
Can we make this a one-liner, perhaps?

E.g. sudo dnf install subversion rpm-build